### DISCUSSION TITLE
full energy consumption - energy map ?

### DISCUSSION THREAD

**Discussion by ckiw** [electricitymaps/electricitymaps-contrib#2803] [Ideas]
Hello,

It would be interresting to have the same king of map but for the full energy consumption of a country (per capita or by GDP) and for longer period of time. Do you know if there is a project of this type ?

With this king of map, it would be possible to see that a country is reducing (or not) the emissions globally. And with the same type of arrows to visualise exportations ;) 

Alexis

**Comment by Kongkille** [comment]
Hi @ckiw this is indeed interesting and something we are looking towards. Whether it can be a direct part of electricitymap or not is unsure, but we are looking at getting a historical view at least.

I'll move this to discussions! Thank you :)
